pkg/tests/end_to_end_tests/migrate_test.go

```diff
@@ -23,6 +23,8 @@ import (
 )
 
 func TestMigrate(t *testing.T) {
+	// TODO (james): We should probably just remove this whole test instead of skipping
```

Comment:
Yes I think this test can be deleted. The takeover path will be covered by the upgrade tests. Let's make sure we do have a tests for CheckVersions, if we don't lets add an issue to add it back in.

Reply:
https://github.com/timescale/promscale/issues/1236